simpleviewinc/tap-events-force
# Image-From Updates

**Ticket**: [ZEN-532](https://jira.simpleviewtools.com/browse/ZEN-532)
> Depends on this keg-cli PR: https://github.com/simpleviewinc/keg-cli/pull/31

## Context

* Because of the new `image from updates`, we need to update the contents in the `/containers` to work with the new workflow

## Goal
* To ensure you can still run the same commands when working in a container
* Updating the containers folder to work with the new image workflow

## Updates

* `/containers/*`
   * Updated the container files to work with the Keg-CLI's `Image-From` updates

## Testing

### Setup
* Pull this branch locally
* Ensure your `keg-cli` is on branch `ZEN-463-push-pull-tasks`
* Ensure you don't have any evf images locally

### Test 1
* In the terminal run `keg evf start`
   * It should pull the evf image from `ghcr.io/simpleviewinc/evf:master`
   * It should then start the evf tap
* Navigate to http://evf-zen-463-image-from-updates.local.kegdev.xyz/
   * Ensure the Tap loads as expected

### Test 2
* Make changes to your local `tap-events-force` repo
* Open a new terminal tab and run `keg evf sync components:install:start`
  * It should install and sync local keg components into the docker container
  * Ensure the changes are reflected in the app
